### PR TITLE
Portfolio Setup Pull Request

### PR DESCRIPTION
**Pull request by Techcolony**
This is the Portfolio Setup Mobile First pull request to merge the main and the feature-setup branches

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,171 @@
...
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
```
Comment (pinkmoon25) [inline README.md:36]: - [ ] [OPTIONAL] As it is important to have a professional readme please fix the project name with the current project i.e Portfolio.

File: README.md [hunk 2]
```diff
@@ -1 +1,171 @@
...
+> This project is the portfolio mobile first setup, it is created using HTML and CSS with Linters also implemented. It has addition feature of responsivity to breaking points on various  mobile devices.
+
+## 🛠 Built With <a name="built-with"></a>
+1. HTML5
+2. CSS3
+3. LINTERS
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
```
Comment (pinkmoon25) [inline README.md:47]: - [ ] [OPTIONAL] please remove this placeholder texts as they are for your understanding only.

File: README.md [hunk 3]
```diff
@@ -1 +1,171 @@
...
+    <li><a href="">LINTERS</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```
Comment (pinkmoon25) [inline README.md:63]: - [ ] [OPTIONAL] please remove this placeholder texts as they are for your understanding only.

File: README.md [hunk 4]
```diff
@@ -1 +1,171 @@
...
+after cloning the project to your local machine, you can use your code editor to explore the files
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Saheed Oyelola (Techcolony) <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```
Comment (pinkmoon25) [inline README.md:116]: - [ ] [OPTIONAL] please remove this placeholder texts as they are for your understanding only.

**Review by pinkmoon25 — APPROVED** [review]
Hi @Techcolony , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters :heavy_check_mark: 
GitHub flow :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 4 above)
Comment (Techcolony) [reply README.md:116]: Thanks for the review

**Pull request merged**